manifest.php

```diff
@@ -50,7 +50,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '39.4.1.3',
+    'version'     => '39.4.1.5',
```

Comment:
What is the reason of double version bump?

Reply:
@btamas 

There was a tag issue, `v39.4.1.4` has been created with a wrong version :(
https://github.com/oat-sa/extension-tao-testqti/blob/v39.4.1.4/manifest.php#L53

Instead of rolling back the tag, which could impact other deploys, I suggested skipping it.

Reply:
I see, thx.